deployment/refinery-modules/refinery/manifests/docker_.pp

```diff
@@ -0,0 +1,5 @@
+class refinery::docker_ {
+  class { 'docker':
```

Comment:
Okay, should this be changed throughout our manifests? I was just following the syntax that we had used throughout